support/ci/shared.ps1

```diff
@@ -30,6 +30,10 @@ function Install-Rustup($Toolchain) {
     }
 }
 
+function Rust-Toolchain {
```

Comment:
The idiomatic Powershell naming here would be `Get-RustToolChain`. Ideally you want to only use `verb`s listed in `get-verb`.